### ISSUE TITLE
[HOLD for payment 2025-01-17] Remove "enable your wallet" task from "Get paid back..." and "Chat and split expenses.." onboarding tasks

### ISSUE THREAD

**Issue by danielrvidal** [Expensify/App#52977]
Now that we have the two minute tour, we would like to remove the following tasks from the following flows:
- Remove "Enable your wallet" task from the Get paid back from my employer onboarding tasks 
- Remove "Add personal bank account" from the Chat and split expenses with friends onboarding tasks. 

Here is the Get paid back from my employer onboarding tasks: 
![image](https://github.com/user-attachments/assets/19ba3ff0-f742-4884-b3fc-e41198cdb2c6)

Here are the current Chat and split expenses with friends onboarding tasks
![image](https://github.com/user-attachments/assets/9ed65fbf-652b-4aec-b6b5-23332cde7498)

<details><summary>Issue Owner</summary>Current Issue Owner: @jjcoffee</details>

**Comment by Kalydosos** [comment]
### Proposal
### Please re-state the problem that we are trying to solve in this issue.
Remove "enable your wallet" task from "Get paid back..." and "Chat and split expenses.." onboarding tasks

### What is the root cause of that problem?
Improvement needed

### What changes do you think we should make in order to solve the problem?

1. "Enable your wallet" is already absent from the tasks of "Get paid back from my employer" onboarding menu. Nothing to do then
2. To remove "Add personal bank account" from the Chat and split expenses with friends onboarding tasks, we just need to remove here
https://github.com/Expensify/App/blob/main/src/CONST.ts#L5102-L5116

no BE change needed

### RESULT
![Capture d’écran de 2024-11-23 16-03-47](https://github.com/user-attachments/assets/cc11d20a-7462-4019-b825-b35778674f07)


### What alternative solutions did you explore? (Optional)
None

**Comment by anmurali** [comment]
@jjcoffee - can you please review these proposals so we can move forward?

**Comment by jjcoffee** [comment]
Thanks @NJ-2020! Looks like the fix has been CP'ed now, but there's still `Add personal bank account` instead of `Enable your wallet`.

I think you may have made a mistake in your proposal, with the third code block that needs updating (it repeats the first). Can you get that updated?

**Comment by jjcoffee** [comment]
Assuming the [above](https://github.com/Expensify/App/issues/52977#issuecomment-2498445822) doesn't change anything regarding what we want to do here, I'm happy to go with @NJ-2020's [proposal](https://github.com/Expensify/App/issues/52977#issuecomment-2494642554), as he was first and the general thrust is correct (even if the third code block to update isn't quite right :wink:).

:ribbon::eyes::ribbon: C+ reviewed

**Comment by Kalydosos** [comment]
@jjcoffee @NJ-2020 mentionned a need to fix the BE, which is not also correct imho

**Comment by danielrvidal** [comment]
Why do you need the description for that if we're removing it? I'm not sure I understand. I think we're remove the task and description.

**Comment by jjcoffee** [comment]
@NJ-2020 No, I just meant that the current behaviour was a bit different from the issue description. We haven't agreed to do anything about it. You can proceed with the PR just dealing with removing the tasks as described in the issue.

**Comment by danielrvidal** [comment]
All good @NJ-2020 I'm guessing this should be pretty easy so do you think you can have a PR to review by EOD Wed?

**Comment by jjcoffee** [comment]
Reviewing the PR today!

**Comment by danielrvidal** [comment]
Awesome, thank you!

**Comment by jjcoffee** [comment]
I think @yuwenmemon is back on Monday, so it depends how urgent this is cc @danielrvidal

**Comment by yuwenmemon** [comment]
Merged it!

**Comment by jjcoffee** [comment]
Not sure if we need this, but here's a regression test in case.

## Regression Test Proposal

### Test:


1. Sign in with a new account
2. Select the `Chat and split expenses with friends` option on the onboarding flow
3. Complete the onboarding steps
4. Verify that there is no 'Add personal bank account' task
5. Repeat the above, but select `Get paid back by my employer` and verify the same task is not there

Do we agree 👍 or 👎

**Comment by johncschuster** [comment]
@NJ-2020 I can't find your Upwork profile. Can you please post it here?

**Comment by johncschuster** [comment]
# Payment Summary
Contributor: @NJ-2020 paid $250 via [Upwork](https://www.upwork.com/jobs/~021880276070638310265)
Contributor+: @jjcoffee owed $250 via NewDot

[Upwork Job](https://www.upwork.com/jobs/~021880276070638310265) - Please apply!

**Comment by garrettmknight** [comment]
$250 approved for @jjcoffee

**Comment by johncschuster** [comment]
Payment has been issued! Thanks for your contributions!
